services/QuillLMS/spec/controllers/api/v1/teacher_notification_settings_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Api::V1::TeacherNotificationSettingsController do
+  let!(:teacher) { create(:teacher) }
+  let(:notification_type) { TeacherNotifications::StudentCompletedDiagnostic }
```

Comment:
If you don't want to couple to a specific type, you could [sample](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/spec/models/provider_classroom_user_spec.rb#L22) the subclasses